ecosystem/sep-0024.md

```diff
@@ -83,6 +83,10 @@ SEP-24 lays out many options for how deposit and withdrawal can work. These are
     * If the `/fee` endpoint is enabled, use it for computing fees when you need to show them to the user.
 * **Authentication**
    * Perform [authentication](#authentication) via SEP-10 before hitting those endpoints
+* **Requested Fields**
+   * If the anchor requests personal information in `/info`, and your app has it or wishes to collect it, send the information to the anchor via [SEP-12](sep-0012.md).
+   * Before sending personal information to an anchor, the user should confirm that they wish to share that information with that anchor, and be given the opportunity to decline, and this step will be skipped.
```

Comment:
The statement `, and this step will be skipped` is confusing on first read. I don't think that part of the sentence offers much given the next dot point discusses that it can continue without it. Can we drop these few words?

Reply:
sep-0024.md